charts/orchestrator/templates/_helpers.tpl

```diff
@@ -120,4 +120,21 @@
         {{- end -}}
         {{- .argoCDNamespace -}}
     {{- end -}}
+{{- end -}}
+
+{{- define "get-cluster-version" -}}
+  {{- $v := "" }}
+  {{- $version :=(lookup "config.openshift.io/v1" "ClusterVersion" "" "version") }}
+  {{- range $version.status.history }}
+    {{- if eq .state "Completed" }}
+      {{- $v = (semver .version) }}
+    {{- end }}
+  {{- end }}
+
+  {{- $validMinors := list "4.13" "4.14" "4.15" "4.16" -}}
+  {{- $versionString := printf "%d.%d" $v.Major $v.Minor -}}
+  {{- if not (semverCompare ">=4.13 <=4.16" $versionString) -}}
```

Comment:
Is the upper limit necessary? And I think the [minimum version](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/administration_guide_for_red_hat_developer_hub/assembly-install-rhdh-ocp#proc-install-rhdh-airgapped-environment-ocp-helm_assembly-install-rhdh-ocp) should be 4.12 if we're going to be tailing along with RHDH.
```
You have installed an Red Hat OpenShift Container Platform 4.12 or later.
```

Reply:
it was more for the fact that locking the upper limit means that now we won't be releasing the operator for OCP v4.17.

Reply:
According to the supported list of RHDH, 1.2 is supported only by OpenShift Container Platform 4.13, 4.14, 4.15

https://access.redhat.com/support/policy/updates/developerhub?extIdCarryOver=true&sc_cid=701f2000001Css5AAC

Hence we should update that list according to the supported version by RHDH to prevent installing it on unsupported target cluster.
I'd also say that for the sake of RC builds, we'll need to specify explicitly the index images for either RHDH or for OpenShift Serverless Logic operators.

Reply:
I opened a ticket for RHDH so they're aware of the inconsistency
https://issues.redhat.com/browse/RHIDP-3137